src/components/Navbar/Navbar.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+
+import Navbar from './Navbar.component';
+
+describe('Navbar', () => {
+  test('should contain search input', () => {
+    render(<Navbar />);
+
+    const searchInput = screen.getByPlaceholderText(/Search/i);
+
+    expect(searchInput).toBeInTheDocument();
+  });
+
+  test('should contain search button', () => {
+    render(<Navbar />);
+
+    const searchButton = screen.queryByRole('button');
```

Comment:
You should not use query here unless you are validating for non existence. see: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence
In this case, you can use: `getByRole`